src/v0/destinations/hs/HSTransform-v1.js

```diff
@@ -80,6 +82,21 @@ const processLegacyIdentify = async (message, destination, propertyMap) => {
       )}/${hsSearchId}`;
       response.method = defaultPatchRequestConfig.requestMethod;
     }
+    
+    if (!propertyMap) {
```

Comment:
repeated code should move to common function

Reply:
updated